src/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -4,7 +4,7 @@
 
 internal static partial class Interop
 {
-    internal static class Libraries
+    internal static partial class Libraries
```

Comment:
Was this necessary?  I'm fine with it, but I'm wondering if you added it just for consistency with the other platforms' Libraries types, or if was needed for compilation to actually succeed.

Reply:
At first it seemed to me it failed once when I was trying to build without partial.
Removing now, to check if the build succeeds without.

Reply:
```
13:45:17 D:\j\workspace\windows-TGrou---2a8f9c29\src\Common\src\Interop\Windows\Interop.Libraries.cs(7,27): error CS0260: Missing partial modifier on declaration of type 'Libraries'; another partial declaration of this type exists [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Data.Odbc\tests\System.Data.Odbc.Tests.csproj]
```

Reply:
The problem here wasn't the lack of partial in this file, it's that the test library is including the Unix libraries file regardless of build:
https://github.com/dotnet/corefx/pull/24621/files#diff-b6c7540e3c3103e246b9cc615b1101e8L18

Again, it's fine to have partial here, but when we change things like this we should make sure we understand why it's necessary.

Reply:
Nice catch. Makes sense.